docs/Users_Guide/tc-diag.rst

```diff
@@ -0,0 +1,192 @@
+
+tc_diag configuration file
+--------------------------
+
+The default configuration file for the TC-Diag tool named **TCDiagConfig_default** can be found in the installed *share/met/config/* directory. It is encouraged for users to copy these default files before modifying their contents. The contents of the configuration file are described in the subsections below.
+
+The TC-Diag tool should be configured to filter the input track data (**-deck**) down to the subset of tracks that correspond to the gridded data files provided (**-data**). The filtered tracks should contain data for only one initialization time but may contain tracks for multiple models.
+
+_______________________
+
+.. code-block:: none
+
+  model = [ "GFSO", "OFCL" ];
+  storm_id = "";
+  basin = "";
+  cyclone = "";
+  init_inc = "";
+  valid_beg = "";
+  valid_end = "";
+  valid_inc = [];
+  valid_exc = [];
+  valid_hour = [];
+  lead = [];
+
+  censor_thresh = [];
+  censor_val    = [];
+  convert(x)    = x;
+
+  data  = {
+     field = [
+          {
+             name = "PRMSL";
+             level = ["L0"];
+          },
+          {
+             name = "TMP";
+             level = ["P1000", "P500"];
+          },
+          {
+             name = "UGRD";
+             level = ["P1000", "P500"];
+          },
+          {
+             name = "VGRD";
+             level = ["P1000", "P500"];
+          }
+      ];
+  }
+  regrid = { ... }
+
+The configuration options listed above are common to many MET tools and are described in :numref:`config_options`. The name and level entries in the data dictionary define the data to be processed.  The regrid dictionary defines if and how regridding will be performed.
+
+_______________________
+
+.. code-block:: none
+
+  n_range = 100;
+
+The **n_range** parameter is the number of equally spaced range intervals in the range-azimuth grid.
+
+_______________________
+
+.. code-block:: none
+
+  n_azimuth = 180;
+
+The **n_azimuth** parameter is the number of equally spaced azimuth intervals in the range-azimuth grid. The azimuthal grid spacing is 360 / **n_azimuth** degrees.
+
+_______________________
+
+.. code-block:: none
+
+  max_range_km = 100.0;
+
+The **max_range_km** parameter specifies the maximum range of the range-azimuth grid, in kilometers. If this parameter is specified and not **rmw_scale**, the radial grid spacing will be **max_range_km / n_range**.
+
+_______________________
+
+.. code-block:: none
+
+  delta_range_km = 10.0;
+
+The **delta_range_km** parameter specifies the spacing of the range rings, in kilometers.
+
+_______________________
+
+.. code-block:: none
+
+  rmw_scale = 0.2;
+
+The **rmw_scale** parameter overrides the **max_range_km** parameter. When this is set the radial grid spacing will be **rmw_scale** in units of the RMW, which varies along the storm track.
+
+_______________________
+
+.. code-block:: none
+
+  compute_tangential_and_radial_winds = TRUE;
```

Comment:
Just for clarity, is compute_tangential_and_radial_winds = False; ever used in the TCDiag tool? Do any of the current diagnostics use U/V winds?

Reply:
So this does not actually appear in the TC-Diag config file anymore. I'll clean up the docs after the RC1 release for #2558.